### DISCUSSION TITLE
My schematic only loads the framework

### DISCUSSION THREAD

**Discussion by sdfewfds** [maruohon/litematica#682] [Q&A] [answered]
It prompts me to be in the scope display layer mode and then only load the schematic framework, how to solve it

**Comment by maruohon** [comment, marked as answer]
Do you mean you get the warning message that you are currently in one of the layer modes? Then that is your issue. You are in something like the Single Layer mode, and that layer is then probably set outside of your current placement height range, thus you won't see any blocks, only the outline.

So go to the Render Layers menu (`M + C` -> `Render Layers`) and switch to the `All` mode if you want to see it all at once.

Another option is that you have the schematic rendering disabled with `M + G`.

**Reply by sdfewfds** [reply]
Thank you so much for the great help, I finally solved this problem
